### ISSUE TITLE
ddns-scripts: GoIP.de Update URL changed

### ISSUE THREAD

**Issue by Olim98** [openwrt/packages#6448]
GoIP.de update URL changed for IPv4 to
```
http://www.goip.de/setip?username=[USERNAME]&password=[PASSWORD]&subdomain=[DOMAIN]&ip=[IP]
```

**Comment by diizzyy** [comment]
Can you do a pull request?

**Comment by dibdot** [comment]
fixed in latest snapshot and 18.06 tree with referenced update commit.
